tests/composer-travis-prephp71.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This is a mostly-duplicate file to get around a PHP version. It is introducing some added complexity in the travis setup and inherent maintenance cost in the future. What is the plan for keeping this in sync with the main composer.json file?

Reply:
Yeah. I'm not overly thrilled with this, but I don't have a better solution squared away yet.

Some alternatives would include to do something bananas like `require` the `phpcs-changed` within the precommit.js file, use it, then uninstall it. Composer, by design, doesn't have a way to install a dependency without saving it to `composer.json`, but I can only imagine how silly that will get when we're actually making commits to `composer.json`.

I think this is better than that though, but this is still suboptimal. I'm letting good not be the enemy of done for the moment.

Reply:
Trying a different approach in 57b3097. Let's only have one `composer.json` but remove `phpcs-changed` on unsupported versions. Since we're only touching `require-dev`, it should be suitable for testing still. Let's see what Travis thinks...